.github/CODEOWNERS

```diff
@@ -1 +1,2 @@
+clear
```

Comment:
is there a reason for this introduction?

Reply:
good catch. might be a typo. will remove